asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -418,10 +431,20 @@ def main():
 
     # Parse symlinks in output_dir
     output_dir = Path(args.output_dir)
+    # check that output_dir exists, otherwise create it
     if not output_dir.exists():
         output_dir.mkdir()
```

Comment:
```suggestion
        output_dir.mkdir(parents=True)
```
what do you think? usually not necessary but can be convenient

Reply:
could also just get rid of the check and push that responsibility to `pathlib`:
```suggestion
    output_dir.mkdir(parents=True, exist_ok=True)
```
(and get rid of the `if` statement)